__react/utils/bindings.js

```diff
@@ -0,0 +1,79 @@
+export function includeUnsubAfterForSubscription(message) {
+  if (hasNatsBindings(message) && message.bindings().nats().unsubAfter()) {
+    return `subscribeOptions.max = '${message.bindings().nats().unsubAfter()}';`;
+  }
+  return '';
+}
+export function includeQueueForSubscription(message) {
+  if (hasNatsBindings(message) && message.bindings().nats().queue()) {
+    return `subscribeOptions.queue = '${message.bindings().nats().queue()}';`;
+  }
+  return '';
+}
+
+/**
+ * Does an object have bindings
+ */
+export function hasNatsBindings(obj) {
+  return obj.bindings && obj.bindings.nats;
```

Comment:
Shouldn't it be like this?

```suggestion
  return obj.bindings() && obj.bindings().nats();
```

Reply:
Yea you are right, I should address this 😄

Reply:
👍 There are others too.

Reply:
We don't have something like `nats()` :) https://github.com/asyncapi/parser-js/blob/master/lib/mixins/bindings.js

Reply:
Nope, I used `binding('nats')` instead :)